### ISSUE TITLE
returned a member with a null or empty member name

### ISSUE THREAD

**Issue by albertwoo** [Fable-Fauna/Fable.Flora#14]
I have something like this:
```fsharp
    type SiteCss = Flora.Stylesheet<"src/client/css/bulma.css">
    let classes = String.concat " " >> Class
    let view =
        div 
          [ classes
              [ SiteCss.Any.button
                SiteCss.Any.is.danger ] ]
          [ str "my demo" ]
```
But i got below error:
The provided type 'Flora.Stylesheet,file="src/client/css/bulma.css"+Any+is' returned a member with a null or empty member nameF# Compiler(3001)
The bulma.css is copied from the test project in the repo. I also tested with the official one but still the same issue.
**Version: 0.6.1**

**Comment by musheddev** [comment]
Hi albert, thanks for bringing this to my attention. 

Note to future self because I don't have time now: The error appears because procCss function outputs a `{([is; ], is-)}` This is appears to come from some class between lines 130-175 of bulma.css. I currently suspect that `is-1by1` is not being treated as a complete identifier in the token stream. I also noticed that the `.identifier.identifier` pattern are being treated as two separate classes which I suspect should not be the case.

**Comment by musheddev** [comment]
@albertwoo Published 0.6.2 to fix this issue.
